dynomite-derive/src/lib.rs

```diff
@@ -184,18 +186,74 @@ fn get_to_attribute_map_trait(
     }
 }
 
+/// Get `v` from `#[dynomite(n = v)]`
+fn get_dynomite_name_eq_value_attribute_value_lit(
```

Comment:
I know this is a pretty verbose and ugly name.
Suggestions welcome!

Reply:
I'm much less opinionated about private method names than I am about public ones